internal/core/worker/runtimewatcher/runtime_watcher_worker.go

```diff
@@ -40,7 +40,10 @@ import (
 
 var _ worker.Worker = (*runtimeWatcherWorker)(nil)
 
-const WorkerName = "runtime_watcher"
+const (
+	WorkerName                = "runtime_watcher"
+	MinRoomsToApplyDisruption = 2
```

Comment:
```suggestion
        // Apply disruptions only if the total number of rooms is greater than MinRoomsToApplyDisruption.
        // It prevents a 0% disruption budget that may block node drains entirely. 
	MinRoomsToApplyDisruption = 2
```

and then we update the condition to be `>`  + remove comments elsewhere